### ISSUE TITLE
Throws error on storybook 8

### ISSUE THREAD

**Issue by technbuzz** [eliasm307/storybook-addon-deep-controls#17]
As mentioned here https://github.com/storybookjs/storybook/issues/26031 ,

The error I get as follows 
```
node_modules/storybook-addon-deep-controls/index.d.ts:6:8 - error TS2307: 
Cannot find module '@storybook/types' or its corresponding type declarations.

6 } from "@storybook/types";
```

**Comment by eliasm307** [comment]
@technbuzz I've added @storybook/types as a peer dependency and published v0.6.3, see justification in #18

**Comment by technbuzz** [comment]
Now I get different error on version 0.7.0

```
ERROR in node_modules/.pnpm/storybook-addon-deep-controls@0.7.0_@storybook+addon-controls@8.2.8_@storybook+types@8.2.8_storybook@8.2.8/node_modules/storybook-addon-deep-controls/index.d.ts:9:8 - error TS2300: Duplicate identifier 'Parameters'.

9   type Parameters = {
         ~~~~~~~~~~

  node_modules/.pnpm/@storybook+core@8.2.8/node_modules/@storybook/core/dist/types/index.d.ts:2:371
    2 export { AnnotatedStoryFn, ArgTypes, ArgTypesEnhancer, Args, ArgsEnhancer, ArgsFromMeta, ArgsStoryFn, BaseAnnotations, Canvas, ComponentAnnotations, ComponentId, ComponentTitle, Conditional, DecoratorApplicator, DecoratorFunction, GlobalTypes, Globals, IncludeExcludeOptions, InputType, LegacyAnnotatedStoryFn, LegacyStoryAnnotationsOrFn, LegacyStoryFn, LoaderFunction, Parameters, PartialStoryFn, PlayFunction, PlayFunctionContext, Renderer, SBArrayType, SBEnumType, SBIntersectionType, SBObjectType, SBOtherType, SBScalarType, SBType, SBUnionType, SeparatorOptions, StepFunction, StepLabel, StepRunner, StoryAnnotations, StoryAnnotationsOrFn, StoryContext, StoryContextForEnhancers, StoryContextForLoaders, StoryContextUpdate, StoryFn, StoryId, StoryIdentifier, StoryKind, StoryName, StrictArgTypes, StrictArgs, StrictGlobalTypes, StrictInputType, Tag } from '@storybook/csf';
                                                                                                                                                                                                                                                                                                                                                                                        ~~~~~~~~~~
    'Parameters' was also declared here.

preview compiled with 1 error
```

**Comment by eliasm307** [comment]
@technbuzz I've published 0.7.1, let me know if this resolves your issue

**Comment by eliasm307** [comment]
@technbuzz I'll assume no response means the issue is resolved
